controllers/workspace/devworkspace_controller.go

```diff
@@ -201,9 +211,11 @@ func (r *DevWorkspaceReconciler) Reconcile(req ctrl.Request) (reconcileResult ct
 		componentDescriptions = append([]controllerv1alpha1.ComponentDescription{cheRestApisComponent}, componentDescriptions...)
 	}
 
-	pvcStatus := provision.SyncPVC(workspace, componentDescriptions, r.Client, reqLogger)
-	if pvcStatus.Err != nil || !pvcStatus.Continue {
-		return reconcile.Result{Requeue: true}, pvcStatus.Err
+	if storagelib.NeedsStorage(workspace.Spec.Template) {
+		pvcStatus := provision.SyncPVC(workspace, componentDescriptions, r.Client, reqLogger)
```

Comment:
components are not used there anymore
```suggestion
		pvcStatus := provision.SyncPVC(workspace, r.Client, reqLogger)
```

Reply:
Good point -- I mostly left other code untouched since I want to do a pass over the reconcile loop and remove ComponentDescriptions everywhere at once. They're only needed for che-rest-apis IIRC.

Reply:
Fixed in 1ac4399